spec/descriptions/tagMaintenanceConfiguration.md

```diff
@@ -0,0 +1,23 @@
+### Maintenance Windows
+Maintenance windows allow you to mute alerts.
+
+### Pausing Maintenance Windows
+Maintenance windows can be paused and resumed.
+
+If a maintenance window is paused, alerts will no longer be muted. 
+
+_NOTE: Pausing unmutes alerts. Maintenance Status is separate from pausing a maintenance window._
```

Comment:
It seems that in the note we are repeating the previous sentence. I would also not talk about statuses here.